### PR TITLE
boards.txt.py: add Wemos ESP-WROOM-02 board

### PR DESCRIPTION
**Pull request by rotulet**
Get fixes from https://github.com/esp8266/Arduino/issues/6551#issuecomment-534737671

### CODE REVIEW

**Comment by mcspr** [comment]
Is it really a wemos board?
Shouldn't this be esp wroom-02 instead?

**Comment by rotulet** [comment]
@mcspr good question! I have got the "official" Wemos logo on it. In many places on internet I found the same board branded Wemos but also clones. But I did not find any references on the Wemos web site!

File: tools/boards.txt.py [hunk 1]
```diff
@@ -621,6 +621,21 @@
...
+    ( 'd1_wroom_02', {
+        'name': 'LOLIN(WEMOS) D1 ESP-WROOM-02',
+        'opts': {
+            '.build.board': 'ESP8266_WEMOS_D1WROOM02',
+            '.build.variant': 'd1_mini',
+            },
+        'macro': [
+            'resetmethod_nodemcu',
+            'flashmode_dio',
+            'flashfreq_26',
```
Comment (AronHetLam) [inline tools/boards.txt.py:633]: `flashfreq_26` doesn't exist in macro list, only in a sub element. This causes boardsgen to fail. 
https://github.com/esp8266/Arduino/blob/c681587895c08944bcb4e84ba9ba86a24461f4a6/tools/boards.txt.py#L1092-L1113

Comment (AronHetLam) [reply tools/boards.txt.py:633]: Adding this to the macros list fixes it:
```
'flashfreq_26': collections.OrderedDict([
        ( '.build.flash_freq', '26' ),
]),
```

Comment (rotulet) [reply tools/boards.txt.py:633]: thanks @AronHetLam, I pushed a fix.

**Comment by AronHetLam** [comment]
@rotulet, I'm not sure what exactly are you unsure about, but I'll explain the problem i had.
When I ran the script for the ESP-wroom-02 board i got an exception because there was no entry in the macros dict for `flashfreq_26`. Adding it, as you've done one the fix you pushed, fixed the issue for me, and i was able to generate a boards.txt with the entries needed for ESP-wroom-02.
I ran the scrip as standalone, and didn't look to deep into how it's all put together. As you can see in the issue I referenced this issue from, I had someone with a ESP-wroom-02 board that couldn't get my code compiled correctly.

**Comment by mcspr** [comment]
Simply run the boards script and commit changes to boards.txt and documentation.

Our Check expects that we have the same filesystem contents before and after running it
https://github.com/esp8266/Arduino/actions/runs/3099983428/jobs/5202829401
(unfold the crossed out section, it shows a git-diff output of what boards.txt.py generated and what changed in the repo)

**Comment by rotulet** [comment]
Sorry for the delay...

Thanks @mcspr for the hint! 

@AronHetLam I updated the board.txt file and rebased on master.

**Pull request merged**